src/amo/css/inc/vars.scss

```diff
@@ -14,7 +13,9 @@ $accent-color: #ebebeb;
 $breakpoints: (
   small: '(max-width: 499px)',
   medium: '(min-width: 500px)',
-  large: '(min-width: 720px)'
+  large: '(min-width: 720px)',
+  desktop: only screen and (min-width: 720px),
+  mobile: only screen and (max-width: 719px),
```

Comment:
Using a mobile breakpoint is handy for hiding things only on mobile rather than *showing* them on all other breakpoints, especially if we ever added new ones: we wouldn't have to go back and add them to lists of "all-but-mobile" `@include respond-to` blocks.

I am cool sticking with one way of defining breakpoint. I get these arguments but as we discussed I feel the frequent/usual *intent* of breakpoints is divorced from the recommendation in the abstract. We, ourselves, are always talking *mobile vs desktop* not *small vs large*.

I'd like to have another think about it, ultimately I'm okay with using small/medium/large... but I don't think that's usually how devs or designers are thinking. Even if it's more pure in concept, I actually think it requires a translation when speaking about intent.